stahl/SteelDigitalOntology/.htaccess

```diff
@@ -0,0 +1,60 @@
+# Rewrite engine setup
+RewriteEngine on
+
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^.*/stahl/SteelDigitalOntology$ https://kibubu.github.io/SteelDigitalOntology/ [R=303,L]
```

Comment:
Here you are creating a w3id for: `https://w3id.org/stahl/SteelDigitalOntology/stahl/SteelDigitalOntology` instead of just `https://w3id.org/stahl/SteelDigitalOntology/`. Is this what you intend?

Reply:
Let me check that locally

Reply:
Is w3id's httpd.conf or apache.conf published somewhere? 

I use docker.io/library/httpd:latest for local testing and adjusted my configuration to match the changes introduced by `.travis.yml` but my setup still behaves differently from w3id.

Reply:
that's a question for @davidlehn

Reply:
The apache config isn't available, but it's really trivial and just serves up the repo.  I test locally with something like the following in w3id.localhost.conf that points at a git checkout.  Doesn't test all the security issues, but good enough to check redirects.
```apache
<VirtualHost *:80>
   ServerName w3id.localhost
   DocumentRoot /whatever/w3id.org
   ErrorLog ${APACHE_LOG_DIR}/w3id.localhost-error.log
   CustomLog ${APACHE_LOG_DIR}/w3id.localhost-access.log "%h %l %u %t \"%r\" %>s %b"
   Header always set Access-Control-Allow-Origin "*"
   <Directory /whatever/w3id.org>
      Options FollowSymLinks
      AllowOverride All
      Require all granted
   </Directory>
</VirtualHost>
```